### PR TITLE
Clean julia 0.4

### PR DESCRIPTION
**Pull request by EQt**

### CODE REVIEW

**Comment by mmagnuski** [comment]
adding appveyor would be nice to see if checks pass on windows too

**Comment by tkelman** [comment]
We just need to have a volunteer to run it on their account. I can set up the webhook once a url is provided.

**Comment by EQt** [comment]
I just setup https://ci.appveyor.com/project/EQt/pyjulia 
But I have not used AppVeyor before. Does anyone know how to configure it?

**Comment by tkelman** [comment]
Can you go to the settings, and copy-paste the "webhook url" here? Then I'll enable the webhook from the github side here and we can try triggering a build, see how it goes.

**Comment by EQt** [comment]
If I understood it right, the webhook url is
https://ci.appveyor.com/api/github/webhook?id=vu38lh59skrtal03

In the latest [Build](https://ci.appveyor.com/project/EQt/pyjulia/build/1.0.6/job/y5hbpfxvmxbbvep0,) the tests just run infinitely long... Don't know why.

**Comment by meggart** [comment]
What is the current status of this PR? Master is broken for me and this PR seems to fix it, can this be merged?

**Comment by mmagnuski** [comment]
+1 for merge, but @jakebolewski seems to be MIA.
Maybe @tkelman can merge?
@EQt @benmoran

**Comment by tkelman** [comment]
The history here needs cleaning up, and CI did not run.

**Comment by mmagnuski** [comment]
Actually, [appveyor did run, but didn't run any tests](https://ci.appveyor.com/project/EQt/pyjulia). 
It seems to fail to run tests because it is using `python -m unittest discover` from *C:\MinGW\bin* while `pyjulia` resides in *C:\projects\pyjulia*. So it is missing only a `cd C:\projects\pyjulia`.
I don't know why Travis didn't run. Can you restart Travis for this PR @tkelman ?
@EQt @benmoran

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,12 @@
 language: python
 python:
     - 2.7
-    - 3.4
+    - 3.5
 compiler:
   - clang
+branches:
+  only:
+    - clean-julia-0.4
```
Comment (tkelman) [inline .travis.yml]: delete this

**Comment by Keno** [comment]
Relevant parts were included in #54.

**Pull request closed without merging**